# goal definition
string dock_id
---
uint16 CODE_SUCCESS = 0
# todo: more error codes

uint16 code                # 0 is success, otherwise is error code
string message             # Information message
---
# Feedback
uint16 STATUS_NONE = 0
# todo: more status codes

string status              # Current status of recording
string message             # Information message